src/styles/utilities/_mixins.scss

```diff
@@ -0,0 +1,91 @@
+@mixin oneColumn {
+  @media (min-width: 0) and (max-width: 488px) {
+    .catalog {
+      display: grid;
+      justify-content: center;
+      max-width: 944px;
+      margin: 0 auto;
+      gap: 48px;
+      grid-template-columns: 1;
+
+      &__card {
+        display: grid;
+        justify-content: center;
+        align-items: center;
+        width: 200px;
+        height: 300px;
+        border: 1px solid #ccc;
+        border-radius: 5px;
+      }
+    }
+  }
+}
+
+@mixin twoColumns {
+  @media (min-width: 488px) and (max-width: 768px) {
+    .catalog {
+      display: grid;
+      justify-content: center;
+      max-width: 944px;
```

Comment:
An interesting approach to solving the problem, but your code is duplicated and if the designer changes the data, it will have to be changed in all places. Rewrite the code so that your mixins are inside classes. Fit it everywhere
Example: 
```
.myButton {
  color: blue;
  margin-top: 8px;
  padding: 8px;

   @media (min-width: 488px) {
    color: red;
  }

  @media (min-width: 768px) {
    color: black;
  }

  @media (min-width: 1024px) {
    color: gray;
    padding: 12px;
  }
}
```